dbt_common/clients/_jinja_blocks.py

```diff
@@ -98,22 +98,22 @@ def end_pat(self):
 
 
 class TagIterator:
-    def __init__(self, data):
-        self.data = data
+    def __init__(self, text):
```

Comment:
while we're here and have context, would you mind adding some type annotations? lots to backfill in this repo!

Reply:
sure.

Reply:
Done!

Reply:
thank you thank you! ✨